src/main/java/Duke.java

```diff
@@ -1,10 +1,38 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Task[] list = new Task[100];
+        int listcounter = 0;
+
+        while(true){
+            Scanner sc = new Scanner(System.in);
+            String line = sc.nextLine();
+            if(line.equals("bye")){
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            else if(line.equals("list")){
+                for(int i=0;i<listcounter;i++){
+                    System.out.println((i + 1) + ".[" + list[i].getStatusIcon() + "] " + list[i].description);
+                }
+            }
```

Comment:
Coding standard violation: the else if should be in the same line as the end brackets.
You may refer to the link below:
  https://se-education.org/guides/conventions/java/basic.html